Firebase Chat/Firebase Chat/Constants/Constants.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Constants.swift
+//  Firebase Chat
+//
+//  Created by David Wright on 4/26/20.
+//  Copyright © 2020 David Wright. All rights reserved.
+//
+
+import Foundation
+
+struct K {
+    static let showChatSegue = "ShowChatSegue"
+    static let chatCellIdentifier = "ChatRoomCell"
+    
+    static let testUser = Sender(senderId: "testUserID-555", displayName: "MissingNo.")
+    static let testMessage = Message(text: "Test message", sender: testUser)
+    static let testChat = ChatRoom(title: "Test Chat", messages: [testMessage])
+    
+    struct FIRKeys {
+        static let chatRooms = "chatRooms"
+        static let messages = "messages"
+        static let users = "users"
+    }
+}
```

Comment:
I like how you set up a Constants file to avoid hard-coded strings in your project. However in the future, to avoid SwiftLint issues, instead of using a struct, use an enum instead. Tip: you can still use static let inside of an enum. Works the same.